client/app/components/FormField.tsx

```diff
@@ -43,7 +37,10 @@ const FormField = ({
         id={id}
         type={type}
         fullWidth
-        {...register(name, { required: true })}
+        {...register(
+          name,
+          isRequire === 'false' ? { required: false } : { required: true }
```

Comment:
C'est bizarre d'avoir un string 'false' , passe la prop en boolean et 
remplace `isRequire === 'false' ? ...` -> `isRequire === false ? ...` -> !isRequire ? ...`

Reply:
On peut même faire mieux:
```typescript
{...register(
          name,
          { required: isRequire }
```

Et au passage renommer isRequire and isRequired